sw/device/lib/dif/dif_otbn.c

```diff
@@ -216,7 +216,8 @@ dif_otbn_result_t dif_otbn_get_err_code(const dif_otbn_t *otbn,
   // Ensure that all values returned from hardware match explicitly defined
   // values in the DIF.
   switch (err_code_raw) {
-    case 0:  // kDifOtbnErrorCodeNoError
+    case 0x0:  // kDifOtbnErrCodeNoError
+    case 0x1:  // kDifOtbnErrCodeBadDataAddr
```

Comment:
You should use the enum names in this switch, rather than the raw values. it will still do exactly the same thing, but is easier to keep in sync.